packages/relayer/processor/process_single.go

```diff
@@ -0,0 +1,60 @@
+package processor
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/taikoxyz/taiko-mono/packages/relayer/bindings/bridge"
+	"github.com/taikoxyz/taiko-mono/packages/relayer/pkg/queue"
+)
+
+func (p *Processor) processSingle(ctx context.Context) error {
+	bridgeAbi, err := abi.JSON(strings.NewReader(bridge.BridgeABI))
+	if err != nil {
+		return err
+	}
+
+	receipt, err := p.srcEthClient.TransactionReceipt(ctx, *p.targetTxHash)
+	if err != nil {
+		return err
+	}
+
+	for _, log := range receipt.Logs {
+		topic := log.Topics[0]
+
+		if topic == bridgeAbi.Events["MessageSent"].ID {
```

Comment:
LGTM, but a 
nitpick it will be better to use ParseMessageSent func([reference link](https://github.com/taikoxyz/taiko-client/blob/bbe9ed764e00029f54baec9e7c51664ee36c489b/bindings/gen_guardian_prover.go#L907)).

Reply:
I agree, I can update this PR tomorrow morning to use that instead.

Reply:
done!